motionplan/cBiRRT.go

```diff
@@ -124,15 +125,17 @@ func (mp *cBiRRTMotionPlanner) rrtBackgroundRunner(
 	mp.start = time.Now()
 
 	if rrt.maps == nil || len(rrt.maps.goalMap) == 0 {
+		mp.logger.Info("initing solutions")
```

Comment:
I don't think we should be adding these 2 log statements to production

Reply:
Whoops. I meant to remove that.